Makefile

```
@@ -1,9 +1,10 @@
 NCOMPUTES ?= 2
+PXEBOOT_ISO=
```

Let's set a default of `ipxe.iso`, and we'll use it to create a new target.
```suggestion
PXEBOOT_ISO := ipxe.iso
```

---

Set default name of ipxe.iso